RELEASE.md

```diff
@@ -6,11 +6,12 @@ Please follow the established format:
 - Include the ID number for the related PR (or PRs) in parentheses
 -->
 
-
 ## Bug fixes and other changes
 
 - Fix incorrect rendering of datasets in modular pipelines. (#1439)
 - Fix broken SVG/PNG exports in light theme. (#1463)
+- Fix error when setting globalToolbar prop value as false. (#1351)
+- Fix error error in dropdown component. (#1351)
```

Comment:
can we have 1 release comment like **Fix dataset and global tool bar error with standalone React component** ?